scss/bootstrap/overrides/style/_forms.scss

```diff
@@ -75,6 +89,36 @@ label {
   }
 }
 
+// Style new group of type radio
+.form-inline {
```

Comment:
Yea i just followed bootstrap way of defining inline form groups to save some extra styling

http://getbootstrap.com/css/#forms-inline

so should i remove form-inline and add the inline styling under "form-inline-tabs" ?
```
.form-inline-tabs .form-group {
  display: inline-block;
}
```